package.json

```diff
@@ -1,16 +1,31 @@
 {
     "name": "backseat-debug",
     "displayName": "Backseat Debug",
-    "version": "0.4.5",
+    "version": "0.4.6",
```

Comment:
Don't update the version number manually. `vsce` will automatically update the version when I publish the extension.